js-mentoring-program-master/tasks/json/fetch/fetch.js

```diff
@@ -4,4 +4,15 @@
  * 3. Save only those items, where id < 20
  * DOCS: https://www.npmjs.com/package/node-fetch
  */
-const fetch = require('node-fetch');
\ No newline at end of file
+const fetch = require('node-fetch');
+const fs = require('fs');
+ 
+fetch('https://jsonplaceholder.typicode.com/posts')
+    .then(response => response.json())
+    .then(result => result.filter(item => item.id < 20))
+    .then(response => 
```

Comment:
I would like you to refactor this code to actual ES6 standards use `async&await` instead of `.then&catch`